composer.json

```diff
@@ -47,7 +47,7 @@
         "react/promise": "To use ReactPHP promise adapter"
     },
     "require-dev": {
-        "phpunit/phpunit": "^4.1 || ^5.5 || ^6.0",
+        "phpunit/phpunit": "^5.5 || ^6.0",
```

Comment:
Since we use `TestCase` and not `PHPUnit_Framework_TestCase` we aren't compatible with phpunit 4